ModelLibrary/Model/MDunningRunLine.cs

```diff
@@ -232,14 +232,70 @@ public void SetPayment(MPayment payment)
         /// <param name="payAmt">amount</param>
         /// <param name="openAmt">open</param>
         public void SetPayment(int C_Payment_ID, int C_Currency_ID,
-            Decimal payAmt, Decimal openAmt)
+            Decimal payAmt, Decimal openAmt, int VA027_PostDatedCheck_ID)
```

Comment:
use polymorphism, don't extend method with new parameter. 
while using in some class, give error